src/components/InputField.js

```diff
@@ -557,7 +562,7 @@ class InputField extends React.Component {
             autoFocus={field.focus}
             placeholder={field.placeholder}
             className={field.className}
-            defaultValue={field.defaultValue || ''}
```

Comment:
Wait I missed that, @eloyekunle I'm afraid this can have a lot of side effects for uncontrolled inputs that we have on the platform. Are you sure this is safe?

Reply:
@Betree I removed it only because of this error:
![image](https://user-images.githubusercontent.com/11166933/58252569-56501f00-7d5e-11e9-8198-04f6b946c335.png)

Reply:
I've reverted it.

Reply:
You can probably fix this warning by setting `defaultValue` instead of `value` on `InputField`.

If we can't prove that this change will not break other forms then we should indeed revert it.

Reply:
Setting `value` instead of `defaultValue` was the fix for the [Remove bug](https://github.com/opencollective/opencollective-frontend/pull/1624#discussion_r286536123) you reported.

Reply:
E2E tests `21` and `27` are broken since this change (see https://github.com/opencollective/opencollective-frontend/pull/1624/commits/754c13545682183ab6f7dd8764ce02b6c83fdfe9), there's a good chance that this change is the reason for it.

https://circleci.com/gh/opencollective/opencollective-frontend/22277

Reply:
I'll look into it.

Reply:
Fixed now @Betree